content/applications/inventory_and_mrp/manufacturing/quality_control/quality_alerts.rst

```diff
@@ -0,0 +1,74 @@
+=====================
+Create quality alerts
+=====================
+
+:doc:`Configuring quality control points <quality_control_points>` is a great way to ensure that
+quality checks are performed at routine stages during specific operations. However, quality issues
+can often appear outside of these scheduled checks. Using *Odoo Quality*, users can create quality
+alerts for issues that are not detected by automated processes.
+
+Create quality alerts from the quality module
+=============================================
```

Comment:
Seems redundant to the h1.
Here's my suggestion that differentiates the section and still reflects what's being discussed.
```suggestion
Find and fill out the quality alerts form
=========================================
```